cmd/acra_configui/acra_configui.go

```diff
@@ -107,10 +115,29 @@ func SubmitSettings(w http.ResponseWriter, r *http.Request) {
 
 func index(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Security-Policy", "require-sri-for script style")
-	parsedTemplate, _ := template.ParseFiles(filepath.Join("static", "index.html"))
-	var outConfigParams configParamsYAML
-	configParamsYAML, err := ioutil.ReadFile("acraserver_config_vars.yaml")
-	check(err)
+	tplPath := fmt.Sprintf("./cmd/acra_configui/%s", filepath.Join("static", "index.html"))
+	tplPath, err = utils.AbsPath(tplPath)
+	if err != nil {
+		log.WithError(err).Errorf("no config file[%v]", tplPath)
+	}
+
+	var parsedTemplate *template.Template
+	parsedTemplate, err = template.ParseFiles(tplPath)
+	if err != nil {
+		log.WithError(err).Errorf("err while parsing template - %v", tplPath)
+		syscall.Exit(1)
```

Comment:
Will be better if we will check template files on app start before running service. after app start check and go down if templates not exist. it will be better than go down at runtime

Reply:
done